### ISSUE TITLE
Mirage Nocrypto initialise should be done within Lwt main loop

### ISSUE THREAD

**Issue by avsm** [mirage/ocaml-conduit#67]
```
let _ = Nocrypto_entropy_lwt.initialize ()
```

the result shouldnt be ignored. See #64 for discussion.

**Comment by yomimono** [comment]
I suspect this issue is no longer relevant since Mirage initializes entropy before starting the user application; is this correct?

**Comment by yomimono** [comment]
I've gained additional confidence in my conclusion above since late 2015 and am closing the issue; feel free to reopen if you disagree.
